avm/res/sql/server/main.bicep

```diff
@@ -557,6 +555,7 @@ output privateEndpoints array = [
 import { diagnosticSettingFullType } from 'br/public:avm/utl/types/avm-common-types:0.2.1'
```

Comment:
As a recommendation as per the other type imports from the avm-common-types, please move this import on top of the parameter where it is used. Makes it a bit easier to understand where the type is coming from.
The reamining types are fine were they are. Would just be nice to have the consistency for the common types.

Reply:
It is not used at a parameter, but as a type _within_ the `databasePropertyType`. You cannot move a type import _inside_ a type declaration AFAIK. What would be your suggestion for that?

Reply:
In that case nevermind :) The error was on my part. Thanks for elaborating @peterbud